daemon/daemon_unix.go

```diff
@@ -1757,7 +1757,23 @@ func (daemon *Daemon) setupSeccompProfile() error {
 }
 
 func (daemon *Daemon) useShimV2() bool {
-	return cgroups.IsCgroup2UnifiedMode()
+	env := os.Getenv("MOBY_DISABLE_SHIM_V2")
+	if env == "" {
+		return true
+	}
+	// MOBY_DISABLE_SHIM_V2 is deprecated from its birth.
+	// MOBY_DISABLE_SHIM_V2 is just added as a workaround for potential regression issues.
+	logrus.Warn("MOBY_DISABLE_SHIM_V2 is deprecated and planned to be removed in future releases")
```

Comment:
See other comments; I think we should print this further down, so only in case a valid value was passed.

Perhaps change to;

```suggestion
	logrus.Warn("WARNING: MOBY_DISABLE_SHIM_V2 is intended for debugging purposes, and planned to be removed in a future release")
```

Reply:
updated